compose.yaml

```diff
@@ -50,3 +50,10 @@ services:
     working_dir: /app
     volumes:
       - .:/app
+
+  markdown-lint:
+    image: registry.gitlab.com/pipeline-components/markdownlint:latest
+    command: mdl --git-recurse .
+    working_dir: /fixer
```

Comment:
why not `/app`, like everywhere else?

Reply:
Info in the commit message 😉.

Reply:
John Doe, few months from now, will not have this detail available, and it's not intuitive when looking at code only

Reply:
I still forget that commits are squashed 🤷‍♂️. I rather work with full history. Maybe it would be better to change all volumes to `/fixer` and then it does not matter why it's like this?

Reply:
myself i would go this path

but anything that is not confusing end user will work for me

Reply:
Done in #7549.